src/components/user-table/body/user-table-body.tsx

```diff
@@ -1,18 +1,16 @@
 import { UserRow } from './user-table-row';
 import { Bruker } from '../../../rest/data/bruker';
-import { OrNothing } from '../../../utils/types/ornothing';
 import './user-table-body.less';
 
 interface UserTableBodyProps {
 	brukere: Bruker[];
-	aktivEnhet: OrNothing<string>;
 }
 
-export const UserTableBody = (props: UserTableBodyProps) => {
+export const UserTableBody = ({ brukere }: UserTableBodyProps) => {
 	return (
 		<div role="rowgroup" className="user-table-body">
-			{props.brukere.map((bruker, idx) => (
-				<UserRow idx={idx} bruker={bruker} aktivEnhet={props.aktivEnhet} key={idx} />
+			{brukere.map((bruker, idx) => (
+				<UserRow idx={idx} bruker={bruker} key={idx} />
```

Comment:
Er jo ikke innført nå, men SonarCloud er litt gretten på at vi bruker arrayIndex som key. Det går vel greit å bruke fnr til det 🤔 
```suggestion
				<UserRow idx={idx} bruker={bruker} key={bruker.brukerFnr} />
```

Reply:
God idé!